src/utils/conditionToDatalog.ts

```diff
@@ -399,10 +399,10 @@ const translator: Record<string, Translator> = {
         arguments: [
           { type: "variable", value: source },
           { type: "constant", value: ":block/refs" },
-          { type: "variable", value: target },
+          { type: "variable", value: `${target}-Ref` },
         ],
       },
-      ...getTitleDatalog({ source: target, target }),
+      ...getTitleDatalog({ source: `${target}-Ref`, target }),
```

Comment:
the problem was that we had two different variables using the same name: `[?log :block/refs ?log]`.

Adding the suffix should differentiate